clang/lib/Serialization/ASTWriter.cpp

```diff
@@ -3597,8 +3597,13 @@ class ASTIdentifierTableTrait {
   /// doesn't check whether the name has macros defined; use PublicMacroIterator
   /// to check that.
   bool isInterestingIdentifier(const IdentifierInfo *II, uint64_t MacroOffset) {
-    if (MacroOffset || II->isPoisoned() ||
-        (!IsModule && II->getObjCOrBuiltinID()) ||
+    II->getObjCOrBuiltinID();
```

Comment:
Hi @Endilll, Does this call to getObjCOrBuiltinID do anything? Static verifiers report it 'useless' since it only returns a value and the value is not used.

Reply:
Thank you for reporting this!
No, might be a leftover from debugging sessions (this PR was a pain).
Fixed in 662ef8604268b207910225ecca90daf30a46720b